AWSScout2/services/cloudtrail.py

```diff
@@ -39,11 +43,29 @@ def parse_trail(self, global_params, region, trail):
             for key in ['IsMultiRegionTrail', 'LogFileValidationEnabled']:
                 if key not in trail_config:
                     trail_config[key] = False
-            trail_details = api_clients[region].get_trail_status(Name=trail['TrailARN'])
+            trail_details = api_client.get_trail_status(Name=trail['TrailARN'])
             for key in ['IsLogging', 'LatestDeliveryTime', 'LatestDeliveryError', 'StartLoggingTime', 'StopLoggingTime', 'LatestNotificationTime', 'LatestNotificationError', 'LatestCloudWatchLogsDeliveryError', 'LatestCloudWatchLogsDeliveryTime']:
                 trail_config[key] = trail_details[key] if key in trail_details else None
+
+        trail_config['wildcard_data_logging'] = self.data_logging_status(trail_config['name'], trail_details, api_client)
```

Comment:
`trail_details` may be referenced before assignment if using multi region trails.